setup.py

```diff
@@ -206,7 +206,13 @@ def package_files(prefix: str, directory: str, extension: str):
     'onnxruntime>=1.12.1,<2',
 ]
 
-extra_deps['mlflow'] = ['mlflow>=2.0.1,<3.0']
+extra_deps['mlflow'] = [
+    'mlflow>=2.0.1,<3.0',
+]
+
+extra_deps['mcli'] = [
+    'mosaicml-cli>=0.3.6,<0.4',
```

Comment:
I think there's a few semantic changes at 0.4.0, and I'd rather pin to be safe. I'll loosen after that change set and possibly make this a required dep since mcli will become a lot lighter.... I'd rather be safe for now